src/csvcubed/models/cube/catalog.py

```diff
@@ -5,13 +5,15 @@
 from abc import ABC, abstractmethod
 from dataclasses import dataclass
 from datetime import date, datetime
-from typing import Optional, Union
+from typing import Dict, Optional, Union
```

Comment:
I think some of these imports aren't necessary anymore. The only one that you've added that's relevant seemt o be `ValidatedModel` now.

Reply:
Removed unnecessary imports.